ister_test.py

```diff
@@ -4148,7 +4148,7 @@ def mock_os_unlink(path):
     os.unlink = mock_os_unlink
 
     try:
-        ister.process_kernel_cmdline("foo")
+        ister.process_kernel_cmdline("foo", timeout=0.1)
```

Comment:
Since the structure of the download changed I think this test probably should be dropped as there is just a single exception block for the logic.